projects/plugins/jetpack/modules/carousel/jetpack-carousel.js

```diff
@@ -904,95 +633,19 @@
 
 			disableKeyboardNavigation();
 
-			domUtil.emitEvent( carousel.container, 'jp_carousel.beforeClose' );
+			domUtil.emitEvent( carousel.overlay, 'jp_carousel.beforeClose' );
 
 			restoreScroll();
+			swiper.destroy();
+			carousel.isOpen = false;
 
-			domUtil.fadeOut( carousel.container, function () {
-				// Clear slide data for DOM garbage collection.
-				carousel.slides = [];
-				carousel.currentSlide = undefined;
-				carousel.gallery.innerHTML = '';
-
-				restoreScroll();
-
-				domUtil.emitEvent( carousel.container, 'jp_carousel.afterClose' );
-			} );
-		}
-
-		function setSlidePosition( slideEl, x ) {
-			if ( ! slideEl ) {
-				return;
-			}
-			slideEl.style.transform = 'translate3d(' + x + 'px,0,0)';
-		}
-
-		function getSlideWidth( slide ) {
-			return parseInt( getComputedStyle( slide.el ).width, 10 );
-		}
-
-		function updateSlidePositions() {
-			var current = carousel.currentSlide;
-			var last = carousel.lastSlide;
-
-			var galleryWidth = carousel.gallery.offsetWidth;
-			var currentWidth = getSlideWidth( current );
-
-			var previous = getPrevSlide( current );
-			var next = getNextSlide( current );
-			var previousPrevious = getPrevSlide( previous );
-			var nextNext = getNextSlide( next );
-
-			var left = Math.floor( ( galleryWidth - currentWidth ) * 0.5 );
-
-			setSlidePosition( current.el, left );
-			domUtil.show( current.el );
-
-			// minimum width
-			fitInfo();
-
-			// prep the slides
-			var direction = current && last && last.index < current.index ? 1 : -1;
-
-			if ( carousel.slides.length > 1 ) {
-				// Since we preload the `previousPrevious` and `nextNext` slides, we need
-				// to make sure they technically visible in the DOM, but invisible to the
-				// user. To hide them from the user, we position them outside the edges
-				// of the window.
-				//
-				// This section of code only applies when there are more than three
-				// slides. Otherwise, the `previousPrevious` and `nextNext` slides will
-				// overlap with the `previous` and `next` slides which must be visible
-				// regardless.
-				if ( direction === 1 ) {
-					if ( nextNext !== previous ) {
-						setSlidePosition( nextNext.el, galleryWidth + getSlideWidth( next ) );
-						domUtil.show( nextNext.el );
-					}
-
-					if ( previousPrevious !== next ) {
-						setSlidePosition(
-							previousPrevious.el,
-							-getSlideWidth( previousPrevious ) - currentWidth
-						);
-						domUtil.show( previousPrevious.el );
-					}
-				} else {
-					if ( nextNext !== previous ) {
-						setSlidePosition( nextNext.el, galleryWidth + currentWidth );
-						domUtil.show( nextNext.el );
-					}
-				}
-
-				setSlidePosition(
-					previous.el,
-					Math.floor( -getSlideWidth( previous ) + screenPadding * 0.75 )
-				);
-				domUtil.show( previous.el );
-
-				setSlidePosition( next.el, Math.ceil( galleryWidth - screenPadding * 0.75 ) );
-				domUtil.show( next.el );
-			}
+			// Clear slide data for DOM garbage collection.
```

Comment:
should be able to use the old fadeOut method here, but couldn't get it working for some reason, so just hacked in a manual close of the overall for now - need to come back to this.